superset/assets/src/chart/Chart.jsx

```diff
@@ -148,6 +153,10 @@ class Chart extends React.PureComponent {
     this.props.addFilter(col, vals, merge, refresh);
   }
 
+  itemClick(data){
+    this.props.itemClick(data);
```

Comment:
It would have been nice if have used action here `this.props.action.itemClicked(data)`. We will do it in UIC-722